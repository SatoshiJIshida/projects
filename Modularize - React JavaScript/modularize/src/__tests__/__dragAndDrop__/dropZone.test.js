import React from "react";
import DropZone from "../../components/dragAndDrop/dropZone";

// Library Jest taken from source https://jestjs.io/

describe("Droppable dropZones", () => {
  it("should be defined", () => {
    expect(DropZone).toBeDefined();
  });

  it("should render correctly", () => {
    const dropZone = <DropZone name="dropZone-test" />;
    expect(dropZone).toMatchSnapshot();
  });
});
