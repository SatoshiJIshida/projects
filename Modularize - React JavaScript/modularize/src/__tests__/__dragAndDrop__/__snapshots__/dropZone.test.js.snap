// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Droppable dropZones should render correctly 1`] = `
<DropZone
  name="dropZone-test"
/>
`;
